src/components/calcite-chip/calcite-chip.tsx

```diff
@@ -0,0 +1,71 @@
+import {
+  Component,
+  h,
+  Host,
+  Prop,
+  State,
+  Event,
+  EventEmitter,
+  Element
+} from "@stencil/core";
+
+@Component({
+  tag: "calcite-chip",
+  styleUrl: "calcite-chip.scss",
+  shadow: true
+})
+export class CalciteChip {
+
+  //--------------------------------------------------------------------------
+  //
+  //  Public Properties
+  //
+  //--------------------------------------------------------------------------
+
+  @Prop() value: string = null;
+
+  // --------------------------------------------------------------------------
+  //
+  //  Private Properties
+  //
+  // --------------------------------------------------------------------------
+
+  @Element() el: HTMLElement;
+
+  @State() hidden = false;
+
+  // --------------------------------------------------------------------------
+  //
+  //  Events
+  //
+  // --------------------------------------------------------------------------
+
+  @Event() calciteChipDismiss: EventEmitter;
+
+  // --------------------------------------------------------------------------
+  //
+  //  Private Methods
+  //
+  // --------------------------------------------------------------------------
+
+  closeClickHandler = (event: MouseEvent) => {
+    event.preventDefault();
+    this.hidden = true;
+    this.calciteChipDismiss.emit(this.el);
+  }
+
+  //--------------------------------------------------------------------------
+  //
+  //  Render Methods
+  //
+  //--------------------------------------------------------------------------
+
+  render() {
+    return (
+      <Host hidden={this.hidden}>
```

Comment:
should the parent component maybe be in charge of hiding it? Seems a little odd that it hides itself. Maybe the parent component can just decide to render it or not?

Reply:
Most other components use "active" to control visibility, having this one be "hidden" seems a bit weird in that context. I think having each be explicitly in control of itself kind of makes sense in a filter chip lockup scenario though.